test/verify/check-machines

```diff
@@ -102,8 +102,18 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         # Ignoring just to unbreak tests for now
         self.allow_journal_messages("Failed to get COMM: No such process")
 
-    def toggleVmRow(self, vmName, connectionName='system'):
-        self.browser.click("tbody tr[data-row-id=vm-{0}-{1}] .pf-c-table__toggle button".format(vmName, connectionName)) # click on the row header
+        # HACK: https://launchpad.net/bugs/1802005
+        if m.image == "ubuntu-stable":
```

Comment:
Is this really still the case? ubuntu-stable == ubuntu 20.10 prerelease now, that bug ought to be fixed. If not, we need to reopen it. But we dropped that hack from all other check-machines cases.